lib/msal-browser/docs/resources-and-scopes.md

```diff
@@ -77,7 +80,7 @@ Consider a slightly different case:
   msalInstance.loginPopup(loginRequest);
   // will fail and fallback to an interactive method prompting a consent screen
   msalInstance.acquireTokenSilent(tokenRequest);
-  // will succeed and return an Access Token with scopes "openid", "profile", "User.Read" and "api://<myCustomApiClientId>/My.Scope"
+  // will succeed and return an Access Token with scope "api://<myCustomApiClientId>/My.Scope"
   msalInstance.acquireTokenSilent(anotherTokenRequest);
```

Comment:
In all requests above where multiple resources are requested in the `scopes` field, we should move any pre-consent scopes to `extraScopesToConsent`. I don't think we should be demonstrating the pattern of requesting multiple resources in the `scopes` field (and it may fail on the /token call).

Reply:
done!